src/componentP/Profile.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import {
+  NameRocket,
+  Profile, RocketItems, Rockets, RocketsItem,
+} from '../componentR/styles';
+
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.rock);
+  const missions = useSelector((state) => state.missions);
+  return (
+    <Profile>
+      <div className="joined-missions">
+        <h3>My Missions</h3>
+        {missions.map((record) => (record.joined ? (
```

Comment:
- [ ] The list of all joined missioned should be done using the `filter()` function.